code/modules/clothing/under/jobs/civilian/civilian.dm

```diff
@@ -135,69 +135,69 @@
 	fitted = FEMALE_UNIFORM_TOP
 	can_adjust = FALSE
 
-/obj/item/clothing/under/rank/civilian/lawyer
+/obj/item/clothing/under/rank/civilian/caa
 	desc = "Slick threads."
 	name = "Lawyer suit"
 	can_adjust = FALSE
 
-/obj/item/clothing/under/rank/civilian/lawyer/black
+/obj/item/clothing/under/rank/civilian/caa/black
 	name = "lawyer black suit"
 	icon_state = "lawyer_black"
 	item_state = "lawyer_black"
 
-/obj/item/clothing/under/rank/civilian/lawyer/black/skirt
+/obj/item/clothing/under/rank/civilian/caa/black/skirt
 	name = "lawyer black suitskirt"
 	icon_state = "lawyer_black_skirt"
 	item_state = "lawyer_black"
 	fitted = FEMALE_UNIFORM_TOP
 	supports_variations = DIGITIGRADE_VARIATION_NO_NEW_ICON
 
-/obj/item/clothing/under/rank/civilian/lawyer/female
+/obj/item/clothing/under/rank/civilian/caa/female
 	name = "female black suit"
 	icon_state = "black_suit_fem"
 	item_state = "bl_suit"
 
-/obj/item/clothing/under/rank/civilian/lawyer/female/skirt
+/obj/item/clothing/under/rank/civilian/caa/female/skirt
 	name = "female black suitskirt"
 	icon_state = "black_suit_fem_skirt"
 	item_state = "bl_suit"
 	fitted = FEMALE_UNIFORM_TOP
 	supports_variations = DIGITIGRADE_VARIATION_NO_NEW_ICON
 	can_adjust = FALSE
 
-/obj/item/clothing/under/rank/civilian/lawyer/red
+/obj/item/clothing/under/rank/civilian/caa/red
 	name = "lawyer red suit"
 	icon_state = "lawyer_red"
 	item_state = "lawyer_red"
 
-/obj/item/clothing/under/rank/civilian/lawyer/red/skirt
+/obj/item/clothing/under/rank/civilian/caa/red/skirt
 	name = "lawyer red suitskirt"
 	icon_state = "lawyer_red_skirt"
 	item_state = "lawyer_red"
 	fitted = FEMALE_UNIFORM_TOP
 	supports_variations = DIGITIGRADE_VARIATION_NO_NEW_ICON
 
-/obj/item/clothing/under/rank/civilian/lawyer/blue
+/obj/item/clothing/under/rank/civilian/caa/blue
 	name = "lawyer blue suit"
```

Comment:
Removes it from the path, but not the name, interesting. Guess you don't feel like renaming the icons either :3c